frontend/server/src/Controllers/User.php

```diff
@@ -3623,11 +3629,8 @@ public static function getCoderOfTheMonthDetailsForSmarty(
             'entrypoint' => 'coder_of_the_month',
         ];
     }
-
     /**
-     * @omegaup-request-param mixed $username
-     *
-     * @return array{smartyProperties: array{STATUS_ERROR: string}|array{profile: array{birth_date?: \OmegaUp\Timestamp|null, classname: string, country: null|string, country_id: null|string, email?: null|string, gender?: null|string, graduation_date: string|null, gravatar_92?: null|string, hide_problem_tags?: bool|null, is_private: bool, locale: null|string, name: null|string, preferred_language: null|string, rankinfo: array{name?: null|string, problems_solved?: int|null, rank?: int|null}, scholar_degree?: null|string, school: null|string, school_id: int|null, state: null|string, state_id: null|string, username: null|string, verified?: bool|null}}, template: string}
+     * @return array{smartyProperties: array{STATUS_ERROR: string}|array{payload: ProfileDetailsPayload, title: string }, entrypoint: string}
```

Comment:
El nombre de la anotación es distinto al que pusiste. Además, el formato de `STATUS_ERROR` se utilizaba en smarty, pero como ahora toda la lógica está en `TS` pudieras incluirlo dentro del `payload`
```suggestion
     * @return array{smartyProperties: array{payload: UserProfileDetailsPayload, title: string }, entrypoint: string}
```

Entonces `UserProfileDetailsPayload` quedaría algo así:
```
* @psalm-type UserProfileDetailsPayload=array{statusError?: string, profile: UserProfileInfo, contests: UserProfileContests, solvedProblems: list<Problem>, unsolvedProblems: list<Problem>, createdProblems: list<Problem>, stats: UserProfileStats, badges: list<string>, ownedBadges: list<Badge>}
```